build/package/Installer.DEB.proj

```diff
@@ -22,6 +22,7 @@
       Outputs="$(SdkInstallerFile)" >
 
       <!-- Install Shared Framework Packages -->
+      <Exec Command="sudo dpkg -i $(DownloadedSharedFrameworkDepsInstallerFile)" />
```

Comment:
You need a corresponding uninstall step here, right?

Reply:
Thanks for pointing this out...